core/src/test/java/org/elasticsearch/action/admin/indices/create/CreateIndexIT.java

```diff
@@ -475,4 +476,34 @@ public void testCreateShrinkIndexFails() throws Exception {
         ensureGreen();
         assertHitCount(client().prepareSearch("target").setSize(100).setQuery(new TermsQueryBuilder("foo", "bar")).get(), 20);
     }
+
+    /**
+     * This test ensures that index creation adheres to the {@link IndexMetaData#SETTING_WAIT_FOR_ACTIVE_SHARDS}.
+     */
+    public void testChangeWaitForActiveShardsSetting() throws Exception {
```

Comment:
Not a great name, I'll change it to something more descriptive like `testDefaultWaitForActiveShardsUsesIndexSetting` and for the other one.